Resources/doc/cookbook/custom-properties.md

```diff
@@ -31,3 +31,11 @@ class CustomPropertyListener implements EventSubscriberInterface
     }
 }
 ```
+
+Service definition:
+```yml
+acme.listener.custom_property:
+    class: AcmeBundle\EventListener\CustomPropertyListener
```

Comment:
You should also add a namespace definition to the php code above to make it consistent.